fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraSessionImpl.java

```diff
@@ -53,10 +55,10 @@
 
     public static final String TIMEOUT_SYSTEM_PROPERTY = "fcrepo.session.timeout";
 
-    private final Session session;
+    private final Session jcrSession;
     private final String id;
     private final Instant created;
-    private final Map<String, String> sessionData;
+    private final ConcurrentHashMap<String, String> sessionData;
```

Comment:
If we aren't claiming that this is thread-safe, what's the point of `ConcurrentHashMap`?